### PR TITLE
Bump new version: 0.11.0

### PR DESCRIPTION
**Pull request by frewsxcv**
Just tested the latest changes with Django 1.7 and everything seems to work

### CODE REVIEW

File: CHANGES [hunk 1]
```diff
@@ -1,3 +1,7 @@
+0.10.11
+
+- Better support for Django 1.6 and Django 1.7
```
Comment (mattrobenolt) [inline CHANGES]: We should bump this to 0.11 or something, and say that we've dropped support for older Django too. I think that's worth a larger version bump.

Comment (frewsxcv) [inline CHANGES]: Sounds good

Comment (frewsxcv) [inline CHANGES]: Just force pushed it

**Comment by frewsxcv** [comment]
@mattrobenolt Was there anything else you wanted me to do with this?

**Pull request merged**